xapian-applications/omega/templates/query

```diff
@@ -37,87 +37,96 @@ $if{$opt{topterms},$include{inc/toptermsjs}}
 input[type=image] {
   border: 0px;
   }
+
+$include{styles.css}
 </style>
 <script type="text/javascript"><!--
 function f2(n) {return (n>9?"":"0")+n;}
 function write_date(time_t) {
 var d=new Date();d.setTime(time_t*1000);
 var D=d.getFullYear()+"-"+f2(d.getMonth()+1)+"-"+f2(d.getDate());
 var T=f2(d.getHours())+":"+f2(d.getMinutes())+":"+f2(d.getSeconds());
-document.write("<span title=\""+D+" "+T+"\">Modified:<br><b>"+D+"</b></span>");
+document.write("<span>Modified:<br/><time datetime=\""+D+" "+T+"\">"+D+"</time></span>");
 }
 // -->
 </script>
 </head>
-<body bgcolor="#ffffff">
-<form name=P method=get
+<body>
+<form id="query-form" name=P method=get
 action="$html{$or{$env{SCRIPT_NAME},omega}}" target="_top">
-<center>
-<input id="omega-autofocus" type=search name=P value="$html{$query}" size=65 autofocus>
+${Search Control - shows search button, options and textbox}
+<fieldset class="query-search">
```

Comment:
It seems weird to me to use `query-search` as the class for all of `fieldset` and the two `input`s. You could alternatively just put `query-search` on the `fieldset`, and rewrite the CSS rules targeting the inputs as (for instance) `fieldset.query-search > input[type=search]`.

I'd probably just use `query` as the class. (I'd probably also use `search` as the `id` for the `form` itself, so you'd then be able to target things like `form#search fieldset.query > input[type=submit]`, which avoids the duplication of `form#query-form fieldset.query-search > input.query-search[type=submit]`.